plugins/justify/plugin.js

```diff
@@ -63,7 +63,7 @@
 		}
 
 		this.allowedContent = {
-			'caption div h1 h2 h3 h4 h5 h6 p pre td th li': {
+			'caption div span h1 h2 h3 h4 h5 h6 p pre td th li': {
```

Comment:
A bit late now, but I wonder if our formulas/placeholders/etc would have worked better as divs with `display:inline` instead
this is pretty similar to the other bug I just fixed, ckeditor doesn't consider spans as one of the expected elements in most areas